src/Api/Meta/Response/CreatePaymentRulesResponse.php

```diff
@@ -0,0 +1,16 @@
+<?php declare(strict_types=1);
+/**
+ * This File is part of JTL-Software
+ *
+ * User: avermeulen
+ * Date: 2020-08-17
+ */
+
+namespace JTL\SCX\Client\Channel\Api\Meta\Response;
+
+use JTL\SCX\Client\Response\AbstractResponse;
+
+class CreatePaymentRulesResponse extends AbstractResponse
+{
+
```

Comment:
The AbstractResponse will assume that every response >=200&&<300 is considered as successful. 
But CreatePaymentRulesResponse will provide a 201 on Success. I would suggest to check only for a 201